src/AutorestSwift/Models/Complex/Value.swift

```diff
@@ -29,7 +29,7 @@ import Foundation
 /// Common base interface for properties, parameters and the like.
 class Value: Codable, LanguageShortcut {
     /// the schema of this Value
-    public var schema: Schema
+    public var schema: Schema?
```

Comment:
Shouldn't Value always have a schema? Why `schema` changes to optional here?

Reply:
It should, and yet, in decoding `model-flattening.json` it turned out there were many "required" properties that weren't supplied. I'll try reverting the change and see if it plays nicely now. It's possible those errors were the result of something else.

Reply:
I was able to revert this back to required without issue.